test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -384,6 +384,7 @@ void UcxContext::progress_worker_event()
     epoll_event ev;
 
     if (ucp_worker_progress(_worker)) {
+        for (int i = 1; i < 8 && ucp_worker_progress(_worker); i++);
```

Comment:
1. please use `()` for each statement, like 
`for (int i = 1; (i < 8) && ucp_worker_progress(_worker); i++);`
2. maybe make `8` configurable?

Reply:
It might make little sense to end users. But it would be helpful for debugging NO_RESOURCE failures due to tx queue full for example. If no one oppose it, I will add a parameter for it.